examples/taptree_of_horror/taptree_of_horror.rs

```diff
@@ -0,0 +1,314 @@
+                                    thresh(2, sha256({k}), after({oct_26_morning}))
+                                ),
+                                and(
+                                    thresh(10, pk({a5}), pk({b5}), pk({c5}), pk({d5}), pk({e5}), pk({f5}), pk({h5}), pk({i5}),  pk({j5}), pk({l5}), pk({s5}), pk({x5})),
+                                    after({oct_26_morning})
+                                )
+                            )
+                        ),
+                        or(
+                            and(
+                                thresh(4, pk({a6}), pk({b6}), pk({c6}), pk({e6}), pk({f6}), pk({h6}), pk({i6}), pk({l6})),
+                                thresh(5, pk({d6}), pk({x6}), pk({j6}), ripemd160({g}), after({oct_25_afternoon}))
+                            ),
+                            and(
+                                thresh(4, pk({a7}), pk({b7}), pk({c7}), pk({e7}), pk({f7}), pk({h7}), pk({i7}), pk({l7})),
+                                thresh(5, pk({d7}), pk({s7}), pk({j7}), ripemd160({g}), after({oct_25_afternoon}))
+                            )
+                        )
+                    )
+                )
+            )
+
+        )"
+    )
+    .replace(&[' ', '\n', '\t'][..], "");
+
+    // make sure policy doesn't have any issues
+    let pol = Concrete::<DescriptorPublicKey>::from_str(&pol_str).unwrap();
+    let policy_desc: Descriptor<DescriptorPublicKey> = pol.compile_tr(None).unwrap();
+
+    // Now, using this public descriptor create the script address
+    let derived_descriptor = policy_desc.at_derivation_index(0).unwrap();
+    let _script_address = derived_descriptor.address(Network::Regtest).unwrap();
+    // println!("the receiving address of this script is: {}", _script_address);
+    // println!("\ndescriptor is: {}\n", desc);
+
+    // We assert internal key is the one used in the descriptor
+    match &policy_desc {
+        Descriptor::Tr(tr) => {
+            // println!("internal: {}, eve: {}", tr.internal_key(), eve_xpub);
+            assert!(tr.internal_key() == &internal_xpub);
+        }
+        _ => panic!("internal spending path is not correct"),
+    }
+
+    //////////// MAKE THE SPENDING TX ////////////////
+
+    let secp: &secp256k1::Secp256k1<secp256k1::All> = &secp256k1::Secp256k1::new();
+
+    let tx_in = TxIn {
+        previous_output: bitcoin::OutPoint {
+            txid: "8888888899999999aaaaaaaabbbbbbbbccccccccddddddddeeeeeeeeffffffff"
+                .parse()
+                .unwrap(),
+            vout: 0,
+        },
+        sequence: Sequence(0),
+        //        sequence: Sequence(40),
+        ..Default::default()
+    };
+
+    let prev_amount = Amount::from_sat(100_000_000);
+    let witness_utxo = TxOut {
+        value: prev_amount, // 10 BTC, has to be exactly the same amount as what's in the txid being used?
```

Comment:
In 1282d30653a3cb4e188e59d5a3dd8de909aa9477:

Yep, it does.

Reply:
left over comments from a younger me. removing.